tests/Resources/ti.ui.test.js

```diff
@@ -0,0 +1,461 @@
+/*
+ * Appcelerator Titanium Mobile
+ * Copyright (c) 2011-Present by Appcelerator, Inc. All Rights Reserved.
+ * Licensed under the terms of the Apache Public License
+ * Please see the LICENSE included with this distribution for details.
+ */
+/* eslint-env mocha */
+/* globals OS_ANDROID,OS_IOS */
+/* eslint no-unused-expressions: "off" */
+'use strict';
+const should = require('./utilities/assertions');
+const utilities = require('./utilities/utilities');
```

Comment:
<!--
  0 failure: 
  1 warning:  tests/Resources/t...
  
  
  DangerID: danger-id-Danger;
  File: tests/Resources/ti.ui.test.js;
  Line: 12;
-->

- :warning: tests/Resources/ti.ui.test.js line 12 – 'utilities' is assigned a value but never used. (no-unused-vars)